uperf-wrapper/README.md

```diff
@@ -0,0 +1,13 @@
+# Smallfile
```

Comment:
I think you meant to add this to smallfile readme file but added it uperf by mistake ?

Reply:
woops, you're right, nice catch :)